packages/jobs/lib/execution/sync.ts

```diff
@@ -216,12 +220,17 @@ export async function handleSyncSuccess({ nangoProps }: { nangoProps: NangoProps
             provider_config_key: nangoProps.providerConfigKey
         };
 
+        providerConfig = await configService.getProviderConfig(connection.provider_config_key, connection.environment_id);
+        if (!providerConfig) {
+            throw new Error(`Provider config not found for connection: ${connection.connection_id}`);
+        }
+
```

Comment:
was it too much of a pain to pass the providerConfig in NangoProps to avoid having to call the DB when handling the output?

Reply:
Wee could save a few queries to the db by putting all static data in NangoProps, like for instance the full account, environment and providerConfig. It might be easier to do that in another PR though

Reply:
Yeah, I'll land this and log that.